components/Portfolio/Insurance/WithdrawalHistory/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Logo } from '@components/General';
+import { TableHead, TableHeadEnd, TableRow, TableCell, SecondaryCell } from '@components/Portfolio';
+
+const WithdrawalHistory: React.FC = () => {
+    const headings = ['Date', 'Market', 'Amount', 'Transaction Details'];
+
+    const tracers = [
+        {
+            date: '24/04/2021',
+            time: '04:31pm',
+            name: 'TSLA',
+            market: 'TSLA-USDC',
+            amount: 453,
+            details: '0x45...3235',
+        },
+        {
+            date: '24/04/2021',
+            time: '04:31pm',
+            name: 'TSLA',
+            market: 'TSLA-USDC',
+            amount: 453,
+            details: '0x45...3235',
+        },
+        {
+            date: '24/04/2021',
+            time: '04:31pm',
+            name: 'TSLA',
+            market: 'TSLA-USDC',
+            amount: 453,
+            details: '0x45...3235',
+        },
+    ];
+
+    const tableHeadEnd = {
+        width: '500px',
+    };
+
+    return (
+        <>
+            <table>
+                <thead>
+                    <tr>
+                        {headings.map((heading, i) =>
+                            i === 3 ? (
+                                <TableHeadEnd theme={tableHeadEnd}>{heading}</TableHeadEnd>
+                            ) : (
+                                <TableHead>{heading}</TableHead>
+                            ),
+                        )}
+                    </tr>
+                </thead>
+                <tbody>
+                    {tracers.map((tracer, i) => (
+                        <TableRow key={`table-row-${i}`}>
+                            <TableCell>
+                                {tracer.date}
+                                <SecondaryCell>{tracer.time}</SecondaryCell>
```

Comment:
Do you mind refactoring this into its own component in components/General, maybe just call it DateAndTime or something with the date and time passed in props, only because ill need something similar. Other than that LGTM